charts/openmetadata/templates/secrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
We shouldn't be creating the Secret Manifest if user provides `secretRef`. Please add in the condition.